docs/using/backup.md

```diff
@@ -164,6 +164,8 @@ Make regular scheduled backups.
                 - In the second field, select one or more minutes (*00* to *59*).
 
         - *Retention*: How many backups to keep. For unlimited, use `0` (zero).
+        - *Retry mode*: In case of error, either let PMM retry the backup again ("Auto") or leave it to the user ("Manual").
```

Comment:
```suggestion
        - *Retry mode*: In case of error, either let PMM retry the backup again ("Auto") or do it again yourself ("Manual").
```

Reply:
On the principle of "talk to the reader"...

Reply:
Agree, thanks!